superset/assets/stylesheets/superset.less

```diff
@@ -195,25 +197,25 @@ div.widget .chart-header {
   opacity: .2;
 }
 
-div.widget {
-  .slice_container {
-    overflow: hidden;
-  }
+// div.widget {
+//   .slice_container {
+//     overflow: hidden;
+//   }
 
-  .stack-trace-container.has-trace {
-    .alert-warning:hover {
-      cursor: pointer;
-    }
-  }
+//   .stack-trace-container.has-trace {
+//     .alert-warning:hover {
+//       cursor: pointer;
+//     }
+//   }
 
-  .is-loading {
-    .stack-trace-container,
-    .slice_container {
-      opacity: 0.5;
-      position: relative;
-    }
-  }
-}
+//   .is-loading {
+//     .stack-trace-container,
+//     .slice_container {
+//       opacity: 0.5;
+//       position: relative;
+//     }
+//   }
+// }
```

Comment:
Delete?

Reply:
Oops, thought I got that already. Just took it out.